tests/Fixer/FunctionNotation/PhpdocToParamTypeFixerTest.php

```diff
@@ -0,0 +1,290 @@
+                    /**
+                    * @param int $foo
+                    * @param string $bar
+                    * @param Baz $hey
+                    * @param float $tab
+                    * @param bool $baz
+                    */
+                    function my_foo(string $bar, int $foo, bool $baz, float $tab, Baz $hey) {}',
+                '<?php
+                    /**
+                    * @param int $foo
+                    * @param string $bar
+                    * @param Baz $hey
+                    * @param float $tab
+                    * @param bool $baz
+                    */
+                    function my_foo($bar, $foo, $baz, $tab, $hey) {}',
+            ],
+            'non-root class with massive string params' => [
+                '<?php
+                    /**
+                    * @param string $bar
+                    * @param string $baz
+                    * @param string $tab
+                    * @param string $foo
+                    */
+                    function my_foo(string $bar, string $baz, string $tab, string $foo) {}',
+                '<?php
+                    /**
+                    * @param string $bar
+                    * @param string $baz
+                    * @param string $tab
+                    * @param string $foo
+                    */
+                    function my_foo($bar, $baz, $tab, $foo) {}',
+            ],
+            'non-root class with different types of params' => [
+                '<?php
+                    /**
+                    * @param string $bar
+                    * @param int $baz
+                    * @param float $tab
+                    */
+                    function my_foo(string $bar, int $baz, float $tab) {}',
+                '<?php
+                    /**
+                    * @param string $bar
+                    * @param int $baz
+                    * @param float $tab
+                    */
+                    function my_foo($bar, $baz, $tab) {}',
+            ],
+            'non-root class with mixed type of param' => [
+                '<?php
+                    /**
+                    * @param mixed $bar
+                    */
+                    function my_foo($bar) {}',
+            ],
+            'non-root namespaced class' => [
+                '<?php /** @param My\Bar $foo */ function my_foo(My\Bar $foo) {}',
+                '<?php /** @param My\Bar $foo */ function my_foo($foo) {}',
+            ],
+            'root class' => [
+                '<?php /** @param \My\Bar $foo */ function my_foo(\My\Bar $foo) {}',
+                '<?php /** @param \My\Bar $foo */ function my_foo($foo) {}',
+            ],
+            'interface' => [
+                '<?php interface Foo { /** @param Bar $bar */ function my_foo(Bar $bar); }',
+                '<?php interface Foo { /** @param Bar $bar */ function my_foo($bar); }',
+            ],
+            'invalid void param on ^7.1' => [
```

Comment:
You say `^7.1` however you pass no version (contrary to the next case). One of them has got to be wrong.